ReactSkia/core_modules/RSkDeviceInfo.cpp

```diff
@@ -45,5 +50,29 @@ jsi::Value RSkDeviceInfoModule::getConstants(
   return jsi::valueFromDynamic(rt, folly::dynamic::object("Dimensions", std::move(dimension)));
 }
 
+void RSkDeviceInfoModule::handlewindowDimensionEventNotification() {
+    SkSize screenSize = RnsShell::PlatformDisplay::sharedDisplay().screenSize();
```

Comment:
Part of this code is common between this function and getConstants function. Can this logic moved to seperate function, to avoid duplicate code